geth/node/status_node.go

```diff
@@ -97,6 +104,28 @@ func (n *StatusNode) start(config *params.NodeConfig) error {
 		n.log.Error("Failed to create an RPC client", "error", err)
 		return RPCClientError(err)
 	}
+	if ethNode.Server().DiscV5 != nil {
+		statusDB, err := db.CreateDatabase(filepath.Join(n.config.DataDir, params.StatusDatabase))
```

Comment:
I would extract this to a separate method, like `startPeerPool`. this `start` method is big already.